### PR TITLE
Cogmap 1 modernisations 2 + Item Offsets overhaul

### LINKED ISSUE #2750: Rad Storm Bug
Submitter: rgbdeadsilent
**Describe the bug**
Radiation storm hit cog1, I went into maintenance above medbay, I stood on the air vent directly above the left APC (Of the set of three APCs). My stats showed I had no debuffs (Like, a radiation timer.) yes I was taking damage, twitching, and mutating. I dunno, doesn't seem right. Died for it, couldn't finish my round out.

**To Reproduce**
1. Buy a Pizza from a vending machine
2. Eat the pizza
3. The pizza has not disappeared
Described above.

**Expected behavior**
Stand in maintenance during rad storm, not dying.

**Screenshots**

**Additional context**

Reported on: Goonstation 2020-11-19 16:55:55

### PR DESCRIPTION
**Pull request by comradef191**
[MAPPING] [BUG]
## About the PR 
-Fixes #2750 + several other APCs
-Moves warning decal floors over to new system, `/obj/decal/tile_edge/stripe/extra_big`, rather than `/turf/simulated/floor/caution`
-Adds a dedicated area for the supply lobby, rather than using another area manually varedited
-Removed d1&d2 var from all cabling as it is unused
-Moves beepsky into the Arrivals Hallway rather than Arrivals Shuttle; Latejoin Jailbirds now actually have a chance to leave arrivals
-Moved Chief Engineers office into their own area, rather than being powered from Engineering
-Updated a few firedoors that I missed in my previous PR
-Offset every item on tables and such for improved aesthetic and clickability
-Changed AI Core & Armoury to use Plasma Glass WinGrilles, killing off some more of the remaining thindows on this map.
-Nulled the access vars on the Engineering Doors, fixing some issues such as: Engineers being able to waltz into Mechanics.

~~(PLANNED?)-Move lighting to the sticky light system, w/ coloured lighting in departments ala Donut3~~ 
## Why's this needed? 
-Bugs bad
-Makes it easier to update and edit this map
-Makes the map prettier to look at, alongside it easier to click on items
-Why, oh god why did the Supply Lobby use an area on the other side of the station but w/ the `name` Var changed

Some examples of Item offsets, before & after:

Engineering Storage
Tech Storage
Chemistry
(Intercom Spawners moved onto walls for ease of viewing)
Radio Room (Intercoms Moved)

## Changelog

```
(u)comradef191:
(+)Fixed a bug on Cogmap1 where standing under some APCs would act as if you were in another room
(+)The Securitron in Cogmap1 Arrivals no longer resides in the Shuttle, and is now placed infront of it in the Hallway
(*)The Armoury & AI Core now use Plasma Glass windows, rather then Thindows, on Cogmap 1.
(*)Overhauled the placement of items atop tables & racks on Cogmap 1 
```

### CODE REVIEW

**Comment by comradef191** [comment]
Trying to get the issues breaking the map fixed, but mapmerge currently hates me and won't let me commit anything...
image

**Comment by comradef191** [comment]
Figured out how to stop Git Hooks being an asshole and not working at all, hurrah, work continues once more!

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
Oh no, it conflicted again. I promise I'll merge this after this merge conflict resolution.

**Pull request merged**
